sample_challenge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PROMPT_COMMAND="$(pwd)/go-term-adventure $(pwd)/sample_challenge.gta" HISTFILE=$HOME/.advhistory bash
```

Comment:
I just mean if there shouldn't be     
`PROMPT_COMMAND="$(pwd)/go-term-adventure $1 $(pwd)/sample_challenge.gta"`    
or something